source/features/one-click-review-submission.tsx

```diff
@@ -9,11 +9,13 @@ import looseParseInt from '../helpers/loose-parse-int.js';
 
 function addButtons(radios: HTMLInputElement[]): void {
 	const form = radios[0].form!;
-	const container = select('.form-actions', form)!;
+	const container
+		= form.closest('.SelectMenu')?.querySelector('.form-actions')
+		?? select('.form-actions', form)!; // TODO: Drop after September 2023
```

Comment:
The action buttons are no longer inside the `form`, they're now linked via `form` attribute